datumaro/datumaro/cli/contexts/project/__init__.py

```diff
@@ -138,19 +142,23 @@ def build_import_parser(parser_ctor=argparse.ArgumentParser):
 
 def import_command(args):
     project_dir = osp.abspath(args.dst_dir)
```

Comment:
Looks like code duplication. It seems that `import` command should be implemented using `create` command with appropriate arguments.